contracts/evm/contracts/Intents/Permit2OrderLib.sol

```diff
@@ -30,55 +22,57 @@ library Permit2OrderLib {
             "uint256 amount,",
             "string toToken,",
             "uint256 toAmount,",
-            "bytes data,"
+            "bytes data)"
         );
     bytes32 internal constant ORDER_TYPE_HASH = keccak256(ORDER_TYPE);
-    string private constant TOKEN_PERMISSIONS_TYPE = "TokenPermissions(address token,uint256 amount)";
+    string private constant TOKEN_PERMISSIONS_TYPE =
+        "TokenPermissions(address token,uint256 amount)";
     string internal constant PERMIT2_ORDER_TYPE =
-        string(abi.encodePacked("SwapOrder witness)", ORDER_TYPE, TOKEN_PERMISSIONS_TYPE));
-
+        string(
+            abi.encodePacked(
+                "SwapOrder witness)",
+                ORDER_TYPE,
```

Comment:
is it missing open bracket here "SwapOrder witness)" ?

Reply:
It is according to spec